sysutils/asdf/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        asdf-vm asdf 0.10.2 v
+categories          sysutils devel
+license             MIT
+
+maintainers         nomaintainer
```

Comment:
You've submitted a number of ports at this point.  You can set yourself as maintainer if you want, filling in your Github handle and obfuscated email address.

Reply:
Sure, I don't mind maintaining this one. Now that I've seen the maintainership process a few times I'm not too worried about taking on a few more.